astropy/coordinates/tests/test_sky_coord.py

```diff
@@ -1398,8 +1398,8 @@ def test_frame_attr_transform_inherit():
 
 def test_deepcopy():
     c1 = SkyCoord(1 * u.deg, 2 * u.deg)
-    c2 = copy.copy(c1)
-    c3 = copy.deepcopy(c1)
```

Comment:
we could also rename `c4` and `c5` to `c2` and `c3`

Reply:
Updated.